deploy-vm.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Install requirements (docker)
+echo "### Removing existing docker libs"
+sudo apt-get remove docker docker-engine docker.io containerd runc -y
```

Comment:
no need to remove docker if it already exists

Reply:
why is this not resolved?

Reply:
Added a check